examples/contrib/modbus_mapper.py

```diff
@@ -296,9 +349,9 @@ def parse(self, value):
         :returns: The decoder method to use
         """
         tokens = self.tokenize(value)
-        token = tokens.next().lower()  # pylint: disable=no-member
+        token = next(tokens).lower()  # pylint: disable=no-member
```

Comment:
```suggestion
        token = tokens.next().lower()  # pylint: disable=no-member
```
Please use our standard.

Reply:
If I used that, python3 crashed.